### ISSUE TITLE
[Feature] Half layer copycats against vertical surfaces

### ISSUE THREAD

**Issue by PaleAsphodel** [copycats-plus/copycats#120]
**Is your feature request related to a problem? Please describe.**
I was going to embellish my doorframe with some extra depth by replacing the copycat step with a slightly less thick half layer copycat, when I realized you couldn't place half layers on vertical surfaces! 
I first thought the rotation was just off, but after using a tool to cycle through its states I realized it was indeed not possible.
The normal layer copycat block can be placed and stacked against vertical surfaces just fine (in fact, my entire hallway uses that), it'd be need if the half layer had the same orientation possibilities!

**Describe the solution you'd like**
Making it possible to place the half layer copycat against vertical surfaces. Since the half layer itself could be placed both horizontally or vertically against a vertical surface, I understand it's twice as much work to implement this as for the full layer copycat.

**Describe alternatives you've considered**
A step can be used, but doesn't have the same level of depth.
A horizontal slice can be used, but doesn't allow the same kind of geometry (in this specific case, it would either not align with the top or the bottom of the doorframe).

**Additional context**
Screenshot 1: doorframe using a simple step for the top
Screenshot 2: failed attempt to use a half layer for the top - the idea was to put it vertically against the wall, and then stack it up until it's almost as thick as a step
![2024-07-24_21 11 43](https://github.com/user-attachments/assets/bbac4d3c-9001-42ba-bbfa-6c47097092dc)
![2024-07-24_21 11 56](https://github.com/user-attachments/assets/d0c5a0fe-b55a-4b01-a4de-6c7b02e38d07)

Thank you!

**Comment by 5h1b** [comment]
Sorry for replying on a old issue, but is this the same as issue [15](https://github.com/copycats-plus/copycats/issues/15)?

**Comment by hlysine** [comment]
Yes it is, and vertical half layers have already been added in v2.2.0 so I'll close this now. Thanks for the reminder.
